### PR TITLE
RR-141 - Retrieve pre-prison qualifications from CIAG

### PR DESCRIPTION
**Pull request by mattwills-moj-digital**
This PR maps the relevant `qualificationsAndTraining` data from the CIAG profile into our `PrePrisonQualifications` view model.

The view template and integration tests will be added in a subsequent PR.

### CODE REVIEW

File: server/@types/viewModels/index.d.ts [hunk 1]
```diff
@@ -128,4 +128,28 @@ declare module 'viewModels' {
   // TODO RR-115 - define the fields
   // eslint-disable-next-line @typescript-eslint/no-empty-interface
   export interface SkillsAndInterests {}
+
+  export interface PrePrisonQualifications {
+    problemRetrievingData: boolean
+    highestEducationLevel?: string
+    targetDateRange?:
```
Comment (nathanrussell-moj-digital) [inline server/@types/viewModels/index.d.ts:135]: targetDateRange ??

File: server/data/mappers/prePrisonQualificationsMapper.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+const toPrePrisonQualifications = (ciagInduction: CiagInduction): PrePrisonQualifications => {
+  if (!ciagInduction || !ciagInduction.qualificationsAndTraining) {
+    return { problemRetrievingData: false, highestEducationLevel: undefined, additionalTraining: undefined }
+  }
+
+  return {
+    problemRetrievingData: false,
+    highestEducationLevel: ciagInduction.qualificationsAndTraining.educationLevel,
+    additionalTraining: ciagInduction.qualificationsAndTraining.additionalTraining,
```
Comment (nathanrussell-moj-digital) [inline server/data/mappers/prePrisonQualificationsMapper.ts:12]: I'm not sure about this - I think there's an array involved isn't there?

File: server/@types/viewModels/index.d.ts (hunk 1 above)
Comment (mattwills-moj-digital) [reply server/@types/viewModels/index.d.ts:135]: Oops, you'd think I copied & pasted this from somewhere! 😂

File: server/data/mappers/prePrisonQualificationsMapper.ts (hunk 2 above)
Comment (mattwills-moj-digital) [reply server/data/mappers/prePrisonQualificationsMapper.ts:12]: Good point, I'll change this.

File: server/testsupport/ciagInductionTestDataBuilder.ts [hunk 3]
```diff
@@ -43,7 +43,31 @@ const aCiagInductionWithPreviousWorkExperience = (prisonNumber = 'A1234BC'): Cia
...
+}
+
+const aCiagInductionWithNoPrePrisonQualifications = (prisonNumber = 'A1234BC'): CiagInduction => {
+  return {
+    ...baseCiagInductionTemplate(prisonNumber),
+    qualificationsAndTraining: null,
+  }
+}
+
+const baseCiagInductionTemplate = (prisonNumber = 'A1234BC'): CiagInduction => {
```
Comment (nathanrussell-moj-digital) [inline server/testsupport/ciagInductionTestDataBuilder.ts:70]: 👍 
A much better name than what I had. Nice 👍

**Review by nathanrussell-moj-digital — APPROVED** [review]
LGTM 👍

**Pull request merged**